apps/system/test/marionette/notification_events_test.js

```diff
@@ -398,22 +393,24 @@ marionette('Notification events', function() {
         type: 'desktop-notification-click',
         id: node.dataset.notificationId
       });
```

Comment:
Now would be a great time to actually perform a real click on the notification container rather than manually dispatching a `mozContentNotificationEvent`. I'm not sure why we didn't do that in the first place TBH.